aem/aem-home.html

```diff
@@ -3,10 +3,10 @@
   <head>
```

Comment:
You should be modifying `index.html`, not `aem-home.html` since this file is the authoritative source of what comes from AEM.